build/linux/toolkit.Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu
+
+# Expose port for the Portainer UI
+EXPOSE 9000
+
+WORKDIR /src/portainer
+
+# Set TERM as noninteractive to suppress debconf errors
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+
+# Install packages
+RUN apt-get update --fix-missing && apt-get install -qq \
+    dialog \
+    apt-utils \
+    curl \
+    build-essential \
+    nodejs \
+    git \
+    wget
+
+# Install Yarn
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+    && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
+    && apt-get update && apt-get -y install yarn
+
+# Install Golang
+RUN cd /tmp \
+    && wget -q https://dl.google.com/go/go1.13.11.linux-amd64.tar.gz \
+    && tar -xf go1.13.11.linux-amd64.tar.gz \
```

Comment:
Introduce a variable via a `ARG` statement for the GO_VERSION

Reply:
Done